kpatch-build/kpatch-build

```diff
@@ -778,6 +787,16 @@ grep -q "CONFIG_DEBUG_INFO_SPLIT=y" "$CONFIGFILE" && die "kernel option 'CONFIG_
 grep -q "CONFIG_GCC_PLUGIN_LATENT_ENTROPY=y" "$CONFIGFILE" && die "kernel option 'CONFIG_GCC_PLUGIN_LATENT_ENTROPY' not supported"
 grep -q "CONFIG_GCC_PLUGIN_RANDSTRUCT=y" "$CONFIGFILE" && die "kernel option 'CONFIG_GCC_PLUGIN_RANDSTRUCT' not supported"
 
+# older versions of pahole don't support extended sections,
+# disable BTF typeinfo generation step in link-vmlinux.sh if needed
+if grep -q "CONFIG_DEBUG_INFO_BTF=y" "$CONFIGFILE" && \
+   ! pahole_btf_version_check; then
+	echo "Backing up scripts/link-vmlinux.s to $TEMPDIR/"
+	cp "$SRCDIR/scripts/link-vmlinux.sh" "$TEMPDIR/"
+	echo "Disabling CONFIG_DEBUG_INFO_BTF in link-vmlinux.s for kpatch-build (incompatible pahole version)"
```

Comment:
I'm thinking these printed messages may be a bit technical and confusing.  How about:

echo "Detected broken version of 'pahole'.  Implementing workaround.'

Though actually maybe we just shouldn't print anything?  kpatch-build tends to print only high-level status messages, and I'm thinking this is really just a minute detail that can probably be ignored.